crates/tests/databases-tests/src/citus/snapshots/databases_tests__citus__schema_tests__schema_test__get_schema.snap

```diff
@@ -3048,5 +3048,169 @@ expression: result
     }
   ],
   "functions": [],
-  "procedures": []
+  "procedures": [
+    {
+      "name": "delete_Album_by_AlbumId",
+      "description": "Delete any value on the public.Album table using the AlbumId key",
+      "arguments": {
+        "AlbumId": {
+          "description": "The identifier of an album",
+          "type": {
+            "type": "named",
+            "name": "int4"
+          }
+        }
+      },
+      "result_type": {
+        "type": "named",
+        "name": "Album"
+      }
+    },
+    {
+      "name": "delete_Artist_by_ArtistId",
+      "description": "Delete any value on the public.Artist table using the ArtistId key",
+      "arguments": {
+        "ArtistId": {
+          "description": "The identifier of an artist",
+          "type": {
+            "type": "named",
+            "name": "int4"
+          }
+        }
+      },
+      "result_type": {
+        "type": "named",
+        "name": "Artist"
+      }
+    },
+    {
+      "name": "delete_Customer_by_CustomerId",
+      "description": "Delete any value on the public.Customer table using the CustomerId key",
+      "arguments": {
+        "CustomerId": {
+          "description": "The identifier of customer",
+          "type": {
+            "type": "named",
+            "name": "int4"
+          }
+        }
+      },
+      "result_type": {
+        "type": "named",
+        "name": "Customer"
+      }
+    },
+    {
+      "name": "delete_Employee_by_EmployeeId",
+      "description": "Delete any value on the public.Employee table using the EmployeeId key",
+      "arguments": {
+        "EmployeeId": {
+          "type": {
+            "type": "named",
+            "name": "int4"
+          }
+        }
+      },
+      "result_type": {
+        "type": "named",
+        "name": "Employee"
+      }
+    },
+    {
```

Comment:
This is how the generated mutations look in the schema output.

Reply:
:heart: @ `v1_delete_…`